src/task/collection.coffee

```diff
@@ -67,6 +81,7 @@ getModuleInfo = (taskId, cnxUrl = '') ->
   moduleInfo
 
 api.channel.on("task.*.receive.*", update)
+api.channel.on('task.*.receive.failure', checkFailure)
 
```

Comment:
yes, but i think it's better to listen specifically on the errors so that checkFailure only happens for those, like an if/filter :)